### PR TITLE
Fix gRPC BBEs and standard library test case

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
$subject

### CODE REVIEW

File: stdlib-integration-tests/grpc_bidirectional_streaming/tests/grpc_bidirectional_streaming_pb.bal [hunk 1]
```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (kalaiyarasiganeshalingam) [inline stdlib-integration-tests/grpc_bidirectional_streaming/tests/grpc_bidirectional_streaming_pb.bal:1]:
```suggestion
// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: stdlib-integration-tests/grpc_bidirectional_streaming/tests/grpc_bidirectional_streaming_pb.bal [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        "google/protobuf/wrappers.proto":"0A1E676F6F676C652F70726F746F6275662F77726170706572732E70726F746F120F676F6F676C652E70726F746F62756622230A0B446F75626C6556616C756512140A0576616C7565180120012801520576616C756522220A0A466C6F617456616C756512140A0576616C7565180120012802520576616C756522220A0A496E74363456616C756512140A0576616C7565180120012803520576616C756522230A0B55496E74363456616C756512140A0576616C7565180120012804520576616C756522220A0A496E74333256616C756512140A0576616C7565180120012805520576616C756522230A0B55496E74333256616C756512140A0576616C756518012001280D520576616C756522210A09426F6F6C56616C756512140A0576616C7565180120012808520576616C756522230A0B537472696E6756616C756512140A0576616C7565180120012809520576616C756522220A0A427974657356616C756512140A0576616C756518012001280C520576616C756542570A13636F6D2E676F6F676C652E70726F746F627566420D577261707065727350726F746F50015A057479706573F80101A20203475042AA021E476F6F676C652E50726F746F6275662E57656C6C4B6E6F776E5479706573620670726F746F33"
+
+    };
+}
```
Comment (BuddhiWathsala) [inline stdlib-integration-tests/grpc_bidirectional_streaming/tests/grpc_bidirectional_streaming_pb.bal:140]:
```suggestion
}

```

File: stdlib-integration-tests/grpc_client_streaming/tests/grpc_client_streaming_pb.bal [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        "google/protobuf/wrappers.proto":"0A1E676F6F676C652F70726F746F6275662F77726170706572732E70726F746F120F676F6F676C652E70726F746F62756622230A0B446F75626C6556616C756512140A0576616C7565180120012801520576616C756522220A0A466C6F617456616C756512140A0576616C7565180120012802520576616C756522220A0A496E74363456616C756512140A0576616C7565180120012803520576616C756522230A0B55496E74363456616C756512140A0576616C7565180120012804520576616C756522220A0A496E74333256616C756512140A0576616C7565180120012805520576616C756522230A0B55496E74333256616C756512140A0576616C756518012001280D520576616C756522210A09426F6F6C56616C756512140A0576616C7565180120012808520576616C756522230A0B537472696E6756616C756512140A0576616C7565180120012809520576616C756522220A0A427974657356616C756512140A0576616C756518012001280C520576616C756542570A13636F6D2E676F6F676C652E70726F746F627566420D577261707065727350726F746F50015A057479706573F80101A20203475042AA021E476F6F676C652E50726F746F6275662E57656C6C4B6E6F776E5479706573620670726F746F33"
+
+    };
+}
```
Comment (BuddhiWathsala) [inline stdlib-integration-tests/grpc_client_streaming/tests/grpc_client_streaming_pb.bal:120]:
```suggestion
}

```

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

File: stdlib-integration-tests/grpc_bidirectional_streaming/tests/grpc_bidirectional_streaming_pb.bal (hunk 1 above)
Comment (daneshk) [reply stdlib-integration-tests/grpc_bidirectional_streaming/tests/grpc_bidirectional_streaming_pb.bal:1]: Though this is shown as this is not a new file. I moved this file.

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**